cluster/gce/trusty/configure.sh

```diff
@@ -152,9 +152,9 @@ install_kube_binary_config() {
   tar xzf "${kube_home}/${manifests_tar}" -C "${dst_dir}" --overwrite
   readonly kube_addon_registry="${KUBE_ADDON_REGISTRY:-gcr.io/google_containers}"
   if [ "${kube_addon_registry}" != "gcr.io/google_containers" ]; then
-    find "${dst_dir}" -name \*.yaml -or -name \*.yaml.in | \
+    find "${dst_dir}" -maxdepth 1 -name \*.yaml -or -maxdepth 1 -name \*.yaml.in | \
```

Comment:
We recently discovered that this failed for GCI due to the incorrect search path: for GCI it should have been `${dst_dir}/kubernetes`. Does this apply to Trusty as well? A quick check would be to try to start a cluster in `europe-west1-c`. Details in #26363.